products-service/serverless.ts

```diff
@@ -0,0 +1,70 @@
+import { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'products-service',
+    // app and org for use with dashboard.serverless.com
+    // app: your-app-name,
+    // org: your-org-name,
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
+    }
+  },
+  // Add the serverless-webpack plugin
+  plugins: ['serverless-webpack'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    region: 'eu-west-1',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+  },
+  functions: {
+    getProductsList: {
+      handler: 'handler.getProductsList',
```

Comment:
Just FYI. It's not necessary to keep handler.ts in root directory, you can specify path to lambda function itself, e.g. `handler: src/handlers/getAllProducts.getProductsList`

Reply:
For me it a little easier to collect all the handlers in one place and then provide them in serverless.ts file (in order to exclude possible errors in path which is a simple string in serverless.ts)